src/main/java/org/codehaus/mojo/taglist/TagListReport.java

```diff
@@ -70,7 +69,9 @@ public class TagListReport extends AbstractMavenReport {
     @Parameter(property = "sourceFileLocale", defaultValue = "en")
     private String sourceFileLocale;
 
-    /** Default locale used if the source file locale is null. */
+    /**
+     * Default locale used if the source file locale is null.
+     */
     private static final String DEFAULT_LOCALE = "en";
```

Comment:
Please note that the default locale in Doxia 2.0.0 has changed from `Locale#ENGLISH` to `Locale#ROOT`. See `org.apache.maven.doxia.tools.SiteTool.DEFAULT_LOCALE`

Reply:
This code is not changed ... if we need I prefer separate PR

Reply:
Let's leave like this. Needs review for Doxia 2.0.0.

Reply:
done